openquake/hazardlib/gsim/si_2020.py

```diff
@@ -0,0 +1,288 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2014-2021 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`SiEtAl2020SInter`
+               :class:`SiEtAl2020SSlab`
+"""
+import numpy as np
+from openquake.baselib.general import CallableDict
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, PGV, SA
+
+
+get_base_term = CallableDict()
+
```

Comment:
Using the CallableDict is fine, but nothing is wrong with a good old if:

def get_base_term(trt, C):
    return C["e"] + (C["d0"] if trt == const.TRT.SUBDUCTION_INTERFACE else C["d1"])